UnattendResources/Logon.ps1

```diff
@@ -75,9 +75,16 @@ function Clean-UpdateResources {
     Remove-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Winlogon" -Name AutoLogonCount -ErrorAction SilentlyContinue
 
     # Cleanup
-    Remove-Item -Recurse -Force $resourcesDir
+    Remove-Item -Recurse -Force -Exclude "sdelete.exe" $resourcesDir
     Remove-Item -Force "$ENV:SystemDrive\Unattend.xml"
-
+    $sdelete = "$resourcesDir\sdelete.exe"

```

Comment:
Let s create a config.psm1 flag for sdelete enable.

I suggest to add sdelete_cleanup=False as default, grouped in section DEFAULT  in the config.psm1 and make the subsequent checks in the code in the logon.ps1 and .psm1 code.

Reply:
The only problem I see here is that the user could set sdelete_cleanup flag on true and for some reason (like a bug in the code) sdelete.exe is not present and the image is presented as a best effort, but sdelete has not been run.
But I can add the extra check in another patch, to make sure that sdelete will be run.